src/test/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositAppSpec.scala

```diff
@@ -145,6 +148,8 @@ class SplitMultiDepositAppSpec extends TestSupportFixture with MockFactory with
         // skipping the Bagging-Date which is different every time
         bagInfo.read().lines.toSeq should contain allElementsOf
           expBagInfo.read().lines.filterNot(_ contains "Bagging-Date").toSeq
+
+        bagInfo.read().lines.filterNot(_ contains "Bagging-Date").toSet should contain theSameElementsAs expBagInfo.read().lines.filterNot(_ contains "Bagging-Date").toSet
```

Comment:
I meant to replace the test above (line 149-150) with this test, rather than adding it